pir/proto/payload.proto

```diff
@@ -40,3 +40,22 @@ message Response {
   // Reply to query as a set of 1 or more serialized ciphertexts.
   Ciphertexts reply = 1;
 }
+
+// BatchRequest sent from the client to the server. Includes an array of
+// ciphertexts, a set of galois keys and relinearization keys to be used.
+message BatchRequest {
```

Comment:
Why create new messages instead of expanding the existing ones? This seems like duplication.